STLtoDATTranslator/Header Files/Triangle.h

```diff
@@ -7,3 +7,6 @@ class Triangle {
     Triangle(Point p1, Point p2, Point p3);
     ~Triangle();
 };
```

Comment:
same make data members private

Reply:
points can be set to private and you can use get methods to access them